content/millennium/r4/base/management/encounter.md

```diff
@@ -225,6 +227,12 @@ Create an individual Encounter.
 
     POST /Encounter
 
+_Implementation Notes_
```

Comment:
Let's move these notes down to the Location specific Notes in the Body fields section.

Reply:
moved https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-3d460d456f4275cf030a4b8eb00128a8f017087dd7a2918741e96202c45e6c52L230 and https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cR550